apps/nextra/pages/en/build/guides/your-first-nft.mdx

```diff
@@ -1,768 +1,453 @@
 
-- `APTOS_NODE_URL`
-- `APTOS_FAUCET_URL`
-</Callout>
-  </Tabs.Tab>
-  </Tabs>
-
----
+```bash filename="Terminal"
+npm install @aptos-labs/ts-sdk dotenv
+npm install --save-dev @types/node
+```
 
-### Step 4.2: Creating local accounts
+### Create tsconfig.json
 
-The next step is to create two accounts locally. [Accounts](../../network/blockchain/accounts.mdx) consist of a public address and the public/private key pair used to authenticate ownership of the account. This step demonstrates how to generate an Account and store its key pair and address in a variable.
+1. Create a new TypeScript configuration file:
 
-<Tabs items={["TypeScript", "Python"]}>
+<Tabs items={["Mac/Linux", "Windows"]}>
   <Tabs.Tab>
-
-```ts filename="example.ts"
-const alice = Account.generate();
-const bob = Account.generate();
-```
-
+    ```bash filename="Terminal"
+    touch tsconfig.json
+    ```
   </Tabs.Tab>
   <Tabs.Tab>
-
-```python filename="example.py"
-alice = Account.generate()
-bob = Account.generate()
-```
-
+    ```bash filename="Terminal"
+    type nul > tsconfig.json
+    ```
   </Tabs.Tab>
 </Tabs>
 
-<Callout type="info">
-Note that this only generates the local keypair. After generating the keypair and public address, the account still does not exist on-chain.
+2. Open the `tsconfig.json` file and add the following configuration:
+
+```json filename="tsconfig.json"
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "types": ["node"],
+    "lib": ["es2020"]
+  }
+}
+```
+
+<Callout type="warning">
+Remember to save your file after adding the configuration!
 </Callout>
 
----
+This configuration ensures TypeScript properly recognizes Node.js types and provides appropriate type checking.
 
-### Step 4.3: Creating blockchain accounts
+### Configure Environment Variables
 
-In order to actually instantiate the Account on-chain, it must be explicitly created somehow. On devnet, you can request free coins with the Faucet API to use for testing purposes. This example leverages the faucet to fund and inadvertently create Alice and Bob's accounts:
+1. Create a `.env` file by running:
 
-<Tabs items={["TypeScript", "Python"]}>
+<Tabs items={["Mac/Linux", "Windows"]}>
   <Tabs.Tab>
-
-```ts filename="example.ts"
-await aptos.fundAccount({
-  accountAddress: alice.accountAddress,
-  amount: 100_000_000,
-});
-await aptos.faucet.fundAccount({
-  accountAddress: bob.accountAddress,
-  amount: 100_000_000,
-});
-```
-
+    ```bash filename="Terminal"
+    touch .env
+    ```
   </Tabs.Tab>
   <Tabs.Tab>
-
-```python filename="example.py"
-bob_fund = faucet_client.fund_account(alice.address(), 100_000_000)
-alice_fund = faucet_client.fund_account(bob.address(), 100_000_000)
-```
-
+    ```bash filename="Terminal"
+    type nul > .env
+    ```
```

Comment:
Can we skip this? I think we should assume that the user knows how to make a new file on their machine. Ideally it just looks like "Create a `.env` file with the following:"